shenyu-plugin/shenyu-plugin-logging/shenyu-plugin-logging-console/pom.xml

```diff
@@ -32,6 +32,11 @@
             <artifactId>shenyu-plugin-base</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.shenyu</groupId>
+            <artifactId>shenyu-plugin-logging-common</artifactId>
+            <version>${project.version}</version>
```

Comment:
why use this?

Reply:
reuse DataMaskByCharReplace、DataMaskByMD5、DataMaskInterface、KeyWordMatch